static/js/unread_ui.js

```diff
@@ -14,6 +14,22 @@ import * as unread from "./unread";
 import {notify_server_messages_read} from "./unread_ops";
 
 let last_mention_count = 0;
+let user_closed_mark_as_read_turned_off_banner = false;
```

Comment:
Does this initial value have any effect, given that `initialize` resets it? I think it'd be better for eventual type-checking reasons to leave this one here, but maybe we can cut the line in `initialize`?

Reply:
I wasn't confident enough that I caught every edge case of ways the banner could end up disappearing, such that it felt good to not rely on `reset` being called each time. If you feel quite confident that `reset_mark_as_read_turned_off_banner` will be called before every `initialize` (except the first) then I could take it out of initialize.

Reply:
It's only ever initialized once, during the general page initialization that happens once per browser load:
```
$ git grep unread_ui.initia
static/js/ui_init.js:    unread_ui.initialize();
```

Reply:
Removed the code to set it to false inside `initialize`, since it's always a `noop`.